res/menu/contact_list.xml

```diff
@@ -2,7 +2,12 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
       xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <item android:title="@string/menu_delete_chat"
```

Comment:
this is not related to the pr, but it turned out the tooltip delete_chat is just wrong for deleting contacts :)